### PR TITLE
Add generate_repo_yaml & init_file_structure to hub_tools

### PR DESCRIPTION
**Pull request by shiyu22**
- When creating the repo, it will auto initialize the file structure with the template.
- Generate the YAML content of Operators according to the function's annotations.

### CODE REVIEW

**Comment by shiyu22** [comment]
In the pylint warning, actually the _field_defaults is the attribute of a Namedtuple, for example, we can get the Outputs info with it:

```bash
>>> from resnet_image_embedding import ResnetImageEmbedding
>>> call = ResnetImageEmbedding.__call__.__annotations__
>>> call
{'img_tensor': <class 'torch.Tensor'>, 'return': <class 'resnet_image_embedding.Outputs'>}
>>> re = call['return']
>>> re
<class 'resnet_image_embedding.Outputs'>
>>> re._field_types
{'feature_vector': <class 'numpy.ndarray'>}
```

So can you update the pylint?

File: towhee/hub/hub_tools.py [hunk 1]
```diff
@@ -385,6 +388,126 @@ def download_files(user: str, repo: str, branch: str, file_list: List[str], lfs_
...
+    init_args = cls.__init__.__annotations__
+    try:
+        del init_args['return']
+    except KeyError:
+        pass
+
+    call_func = cls.__call__.__annotations__
+    try:
+        call_output = call_func.pop('return')
+        call_output = call_output._field_types
```
Comment (fzliu) [inline towhee/hub/hub_tools.py:490]: Try `__annotations__` here.

**Review by fzliu — CHANGES_REQUESTED** [review]

File: towhee/hub/hub_tools.py (hunk 1 above)
Comment (shiyu22) [reply towhee/hub/hub_tools.py]: Thanks, it works and is updated.

**Pull request merged**